app/tasks/maintenance/update_api_entreprise_token_expires_at_task.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Maintenance
+  class UpdateAPIEntrepriseTokenExpiresAtTask < MaintenanceTasks::Task
+    def collection
+      Procedure.where.not(api_entreprise_token: nil)
```

Comment:
```suggestion
      Procedure.with_discarded.where.not(api_entreprise_token: nil)
```

Pour l'appliquer aussi aux démarches supprimées, qui peuvent être restaurées